lamindb/curators/core.py

```diff
@@ -205,6 +205,69 @@ def save_artifact(
         # constructor signature
         pass  # pragma: no cover
 
+    def __repr__(self) -> str:
+        from lamin_utils import colors
+
+        if self._schema is not None:
+            # Schema might have different attributes
+            if hasattr(self._schema, "name") and self._schema.name:
```

Comment:
Better is the below, I believe, I just saw this in code from @Koncopd -- @sunnyosun & I have always done this the wrong way.

```python
if getattr(self._schema, "name", False):
```